File: php/async/AsyncSymbolContext.php
<?php

namespace ccxt;

class AsyncSymbolContext {
    public function __construct ($conxid){
        $this->conxid = $conxid;
        $this->subscribed = FALSE;
        $this->subscribing = FALSE;
        $this->data = array();
    }
    
    public function reset () {
        $this->subscribed = FALSE;
        $this->subscribing = FALSE;
        $this->data = array();
    }
}

File: python/ccxt/async/websocket/async_symbol_context.py
# -*- coding: utf-8 -*-


class AsyncSymbolContext:
    def __init__(self, conxid):
        self.subscribed = False
        self.subscribing = False
        self.data = {}
        self.conxid = conxid

    def reset(self):
        self.subscribed = False
        self.subscribing = False
        self.data = {}
